ArduPlane/Parameters.cpp

```diff
@@ -1339,6 +1315,26 @@ const AP_Param::ConversionInfo conversion_table[] = {
     { Parameters::k_param_fence_autoenable,   0,      AP_PARAM_INT8, "FENCE_AUTOENABLE"},
 };
 
+const AP_Param::ConversionInfo rc_chan_params[] = {
+    { Parameters::k_param_flapin_channel_old, 0, AP_PARAM_INT8, nullptr},
+    { Parameters::k_param_g2, 968, AP_PARAM_INT8, nullptr},
+    { Parameters::k_param_fence_channel, 0, AP_PARAM_INT8, nullptr},
+    { Parameters::k_param_reset_mission_chan, 0, AP_PARAM_INT8, nullptr},
+    { Parameters::k_param_parachute_channel, 0, AP_PARAM_INT8, nullptr},
+    { Parameters::k_param_fbwa_tdrag_chan, 0, AP_PARAM_INT8, nullptr},
+    { Parameters::k_param_reset_switch_chan, 0, AP_PARAM_INT8, nullptr},
+};
+
+const RC_Channel::AUX_FUNC rc_option_values[] = {
+    RC_Channel::AUX_FUNC::FLAP,
+    RC_Channel::AUX_FUNC::SOARING,
+    RC_Channel::AUX_FUNC::FENCE,
+    RC_Channel::AUX_FUNC::MISSION_RESET,
+    RC_Channel::AUX_FUNC::PARACHUTE_RELEASE,
+    RC_Channel::AUX_FUNC::FBWA_TAILDRAGGER,
+    RC_Channel::AUX_FUNC::MODE_SWITCH_RESET,
+};
```

Comment:
static assert for 2 tables of same length

Reply:
even better as one table

Reply:
fixed